README.md

```diff
@@ -3,11 +3,758 @@
 [![Build
 Status](https://ci.jenkins.io/job/Plugins/job/cobertura-plugin/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/cobertura-plugin/job/master/)
 
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/cobertura.svg)](https://plugins.jenkins.io/cobertura)
+
+<!-- Commenting out the changelog badge as >
+<!-- [![GitHub release](https://img.shields.io/github/release/jenkinsci/cobertura.svg?label=changelog)](https://github.com/jenkinsci/cobertura-plugin/releases/latest) -->
```

Comment:
Hi @dduportal. Thanks for the PR! I'm curious about this commented out code - could we just remove it?

Reply:
Removed it! I forgot to do it when pushing.